src/distPlus/distribution/XYShape.re

```diff
@@ -137,6 +143,67 @@ module XtoY = {
       };
     n;
   };
+
+  /* Returns a between-points-interpolating function that can be used with PointwiseCombination.combine.
+     Interpolation can either be stepwise (using the value on the left) or linear. Extrapolation can be `UseZero or `UseOutermostPoints. */
+  let continuousInterpolator = (interpolation: DistTypes.interpolation, extrapolation: DistTypes.extrapolation): interpolator => {
+    switch (interpolation) {
```

Comment:
I imagine you could simplify this switch statement to not be nested, like
```
switch(interpolation, extrapolation){
  | (`Linear, `UseZero) => 
}
```

Reply:
Good catch, thanks